core/src/index/knowhere/knowhere/index/vector_index/IndexBinaryIDMAP.cpp

```diff
@@ -112,6 +112,22 @@ BinaryIDMAP::QueryById(const DatasetPtr& dataset_ptr, const Config& config) {
 }
 #endif
 
+int64_t
+BinaryIDMAP::Count() {
+    if (!index_) {
```

Comment:
server crash is caused by "index_" not initialized ?
why ?

Reply:
No reason for the problem was found, but it would have prevented the server from crashing. @cydrain